src/registrar/models/website.py

```diff
@@ -14,5 +16,35 @@ class Website(models.Model):
         help_text="",
     )
 
+    @property
```

Comment:
**question (non-blocking)**: It looks like this code is close to duplicated with `domain`. I'm assuming that the abstraction necessary to keep from duplicating wasn't worth it in this case.